### ISSUE TITLE
open edX: Set FROM address for bulk email to mitxonline-support@mit.edu

### ISSUE THREAD

**Issue by briangrossman** [mitodl/mitxonline#117]
### Steps to Reproduce

- Go to bulk email interface in LMS: [example](https://courses-qa.mitxonline.mit.edu/courses/course-v1:ShiraU+14.750x_SF+3T2021/instructor#view-send_email)
- Send a test email to yourself 
- View the from address of the email

### Expected Behavior

- The from address should be: mitxonline-support@mit.edu

### Actual Behavior

- The from address is something like: 14.750x_SF-no-reply@edxapp-mail-qa.mitxonline.mit.edu

### Related Issues

- https://github.com/mitodl/ol-infrastructure/issues/327
   - this issue references [settings](https://github.com/mitodl/ol-infrastructure/issues/327#issuecomment-902840954) that need to be set

**Comment by blarghmatey** [comment]
This should be resolved. Can you confirm @briangrossman ?

**Comment by briangrossman** [comment]
@blarghmatey I tried to test, but it looks like it's not sending out messages... here's [the queue](https://courses-qa.mitxonline.mit.edu/admin/instructor_task/instructortask/)

**Comment by blarghmatey** [comment]
Yeah, looks like the issue is the switch to using the mitxonline-support@mit.edu address as the `FROM` field. I have to register that as an identity in SES for it to allow it to send.

**Comment by blarghmatey** [comment]
This should be working now. Relevant changes are in https://github.com/mitodl/ol-infrastructure/pull/349
